app/tests/to-layer/normal/events.patterns

```diff
@@ -0,0 +1,2 @@
+s/.*hid_listener_keycode/kp/p
+s/.*to_keymap_binding/to/p
```

Comment:
Please add some pattern to record all layer-state-changed events, so you can verify which layer state changed events are seen.